librz/core/cbin.c

```diff
@@ -29,6 +29,30 @@
 	if (binfile && binfile->rbin && binfile->rbin->verbose) \
 	eprintf
 
+static RZ_NULLABLE RZ_BORROW const RzList *core_bin_strings(RzCore *r, RzBinFile *file);
+static void _print_strings(RzCore *r, const RzList *list, PJ *pj, int mode, int va);
+static bool bin_raw_strings(RzCore *r, PJ *pj, int mode, int va);
+static int bin_info(RzCore *r, PJ *pj, int mode, ut64 laddr);
+static int bin_main(RzCore *r, PJ *pj, int mode, int va);
+static int bin_dwarf(RzCore *core, PJ *pj, int mode);
```

Comment:
Would it be possible to sort this list somehow?

Reply:
Yes, but these declarations are also only temporary until the are all unfolded and refactored.

Reply:
ok